test/unit/commands/ssl-certificate-specs.js

```diff
@@ -0,0 +1,20 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
```

Comment:
chaiAsPromised import is redundant as we are not dealing with promises in this test module